apps/explorer/src/hooks/useGetEpochs.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '@mysten/core';
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+export const DEFAULT_EPOCHS_LIMIT = 20;
+
+// Fetch transaction blocks
+export function useGetEpochs(limit = DEFAULT_EPOCHS_LIMIT) {
+    const rpc = useRpcClient();
+
+    return useInfiniteQuery(
+        ['get-epochs-blocks', limit],
+        async ({ pageParam }) =>
```

Comment:
Nit, no need for async await.  you can also `({ pageParam = null })` and ` getNextPageParam: ({ nextCursor, hasNextPage }) => hasNextPage ? nextCursor : null`